src/simulators/qasm/qasm_controller.hpp

```diff
@@ -82,6 +82,17 @@ namespace Simulator {
  *      parallel circuit or shot execution is enabled this will only
  *      use unallocated CPU cores up to max_parallel_threads. [Default: 100]
  *
+* From MatrixProductState::State class
+ * - "mps_approximation_threshold" (double): Set the cutoff value for the
+ *      Schmidt coefficients. Numbers smaller than the threshold will be discarded.
+ *      [Default: 1e-16]
+ *
+ * - "mps_approximation_num_sv" (uint): Set the cutoff value for the number
```

Comment:
Can you change the name of this option to `"matrix_product_state_bond_dimension"`, since that is (I believe) the standard naming convention in tensor network community. 


```
*-  "matrix_product_state_bond_dimension" (uint): Set the maximum size of the matrix product state bond dimensions. This sets a cutoff value for the number of Schmidt coeffs...
```